packages/foundation-models/README.md

```diff
@@ -60,26 +60,42 @@ const chatClient = new AzureOpenAiChatClient({
 });
 ```
 
-### Azure OpenAI Client
+### Chat Client
 
-The Azure OpenAI client can then be used to send chat completion or embedding requests to models deployed in the SAP generative AI hub.
+Use the `AzureOpenAiChatClient` to send chat completion requests to an OpenAI model deployed in SAP generative AI hub.
 
-#### Chat Client
+The client sends request with Azure OpenAI API version `2024-06-01`.
+Set a different version together with other request configuration in the `requestConfig` parameter.
 
-Use the `AzureOpenAiChatClient` to send chat completion requests to an OpenAI model deployed in SAP generative AI hub.
+The following example shows a call with messages and optional request configuration.
 
 ```ts
 import { AzureOpenAiChatClient } from '@sap-ai-sdk/foundation-models';
 
 const chatClient = new AzureOpenAiChatClient('gpt-4o');
-const response = await chatClient.run({
-  messages: [
-    {
-      role: 'user',
-      content: 'Where is the deepest place on earth located'
+const response = await chatClient.run(
+  {
+    messages: [
+      {
+        role: 'user',
+        content: 'Where is the deepest place on earth located'
+      }
+    ]
+  },
+  {
+    headers: {
+      'content-type': 'application/json',
```

Comment:
@KavithaSiva already added a similar section in her PR. Please ensure we dont end up with dups

Reply:
This falls under the scope of the issue. I had a look into the clean up PR and see a section documenting custom config only in orchestration. @KavithaSiva I think the easiest way is to remove the section you created in #159